app/plugins/prometheus.md

```diff
@@ -0,0 +1,171 @@
+kong_latency_bucket{type="request",service="google",le="00300.0"} 1
+kong_latency_bucket{type="request",service="google",le="00400.0"} 1
+.
+.
+kong_latency_bucket{type="request",service="google",le="+Inf"} 2
+kong_latency_bucket{type="upstream",service="google",le="00300.0"} 2
+kong_latency_bucket{type="upstream",service="google",le="00400.0"} 2
+.
+.
+kong_latency_bucket{type="upstream",service="google",le="+Inf"} 2
+kong_latency_count{type="kong",service="google"} 2
+kong_latency_count{type="request",service="google"} 2
+kong_latency_count{type="upstream",service="google"} 2
+kong_latency_sum{type="kong",service="google"} 2145
+kong_latency_sum{type="request",service="google"} 2672
+kong_latency_sum{type="upstream",service="google"} 527
+# HELP kong_latency_total Latency added by Kong, total request time and upstream latency aggreggated across all services in Kong
+# TYPE kong_latency_total histogram
+kong_latency_total_bucket{type="kong",le="00001.0"} 1
+kong_latency_total_bucket{type="kong",le="00002.0"} 1
+.
+.
+kong_latency_total_bucket{type="kong",le="+Inf"} 2
+kong_latency_total_bucket{type="request",le="00300.0"} 1
+kong_latency_total_bucket{type="request",le="00400.0"} 1
+.
+.
+kong_latency_total_bucket{type="request",le="+Inf"} 2
+kong_latency_total_bucket{type="upstream",le="00300.0"} 2
+kong_latency_total_bucket{type="upstream",le="00400.0"} 2
+.
+.
+.
+kong_latency_total_bucket{type="upstream",le="+Inf"} 2
+kong_latency_total_count{type="kong"} 2
+kong_latency_total_count{type="request"} 2
+kong_latency_total_count{type="upstream"} 2
+kong_latency_total_sum{type="kong"} 2145
+kong_latency_total_sum{type="request"} 2672
+kong_latency_total_sum{type="upstream"} 527
+# HELP kong_nginx_http_current_connections Number of HTTP connections
+# TYPE kong_nginx_http_current_connections gauge
+kong_nginx_http_current_connections{state="accepted"} 8
+kong_nginx_http_current_connections{state="active"} 1
+kong_nginx_http_current_connections{state="handled"} 8
+kong_nginx_http_current_connections{state="reading"} 0
+kong_nginx_http_current_connections{state="total"} 8
+kong_nginx_http_current_connections{state="waiting"} 0
+kong_nginx_http_current_connections{state="writing"} 1
+# HELP kong_nginx_metric_errors_total Number of nginx-lua-prometheus errors
+# TYPE kong_nginx_metric_errors_total counter
+kong_nginx_metric_errors_total 0
+
+```
+
+In most configurations the Kong Admin API will be behind a firewall or would
+need to be setup to require authentication, here are a couple of options to
+allow access to the `/metrics` endpoint to Prometheus.
+
+
+1. Kong Enterprise users can protect the admin `/metrics` endpoint with an
+   [RBAC user](/enterprise/latest/setting-up-admin-api-rbac) that the
+   Prometheus servers uses to access the metric data. Access through any
+   firewalls would also need to be configured.
+
+2. You can proxy the Admin API through Kong itself, then you can create a route
```

Comment:
This assumes that the proxy port is running on the same node as admin port.

It is possible to disable proxy (data plane) or admin (control plane) in Kong on certain nodes as needed.

Can we mention this along the lines:
`If all Kong nodes are listening for admin and proxy ports, then it is possible to create a route to /metrics endpoint.`

Reply:
I removed the example, and expanded the explanation. This configuration is too big to be a single bullet point, better to link to the relevant documentation.